src/main/java/racingcar/GameManager.java

```diff
@@ -0,0 +1,54 @@
+package racingcar;
+
+import constant.Constants;
+import inspector.InspectRaceResult;
+import java.util.ArrayList;
+import utils.ContentParser;
+
+public class GameManager {
+    private ArrayList<RacingCar> racingCarGroup;
+    private int racingTurn;
+    private int currentTurn = 0;
+
+    public void execute() {
+        playerSetting();
+        raceSetting();
+    }
+
+    private void playerSetting(){
```

Comment:
```suggestion
    private void playerSetting() {
```
다른 부분도 공백을 신경써주시면 좋을 것 같아요~

Reply:
이 부분은 꼼꼼하게 리팩터링을 했다고 해도 절대로 알아차리지 못했을 것 같습니다... 컨벤션에 좀 더 신경 써보겠습니다.